bindings/wasm/examples/src/util.ts

```diff
@@ -81,17 +91,17 @@ export async function ensureAddressHasFunds(client: Client, addressBech32: strin
 
 /** Returns the balance of the given Bech32-encoded address. */
 async function getAddressBalance(client: Client, addressBech32: string): Promise<number> {
-    const outputIds = await client.basicOutputIds([
+    const outputIds: IOutputsResponse = await client.basicOutputIds([
         { address: addressBech32 },
         { hasExpiration: false },
         { hasTimelock: false },
         { hasStorageDepositReturn: false },
     ]);
-    const outputs = await client.getOutputs(outputIds);
+    const outputs = await client.getOutputs(outputIds.items);
 
     let totalAmount = 0;
     for (const output of outputs) {
-        totalAmount += Number(output.output.amount);
+        totalAmount += Number(output.output.getAmount());
```

Comment:
We shouldn't convert a `bigint` to a `number`. Maybe return a `bigint` here instead, arithmetic works on `bigint` just like on number.

Reply:
Getting this issue though:
```
Operator '>' cannot be applied to types 'BigInt' and 'number'.ts(2365)
```

Reply:
you have to convert the number to a bigint using `BigInt(variable)`

Reply:
Yeah getting this error on a `BigInt(number)` value. It's not a conversion problem it's that you cannot use `>` on a bigint.